### PR TITLE
chore: update references to old versions of the host

### PR DESCRIPTION
**Pull request by connorsmith256**
I found some more references to the OTP host

### CODE REVIEW

File: docs/deployment/tracing.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ As you can see, we retrieved the list of vets in our database in just under 40ms
 
 In order to enable tracing we'll take a look at the above example to show you what to set up. At a high level, you'll need:
 
-1. wasmCloud [v0.55.1](https://github.com/wasmCloud/wasmcloud-otp/releases/tag/v0.55.1) or later with the following environment variables:
+1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases) or later with the following environment variables:
```
Comment (vados-cosmonic) [inline docs/deployment/tracing.md]:
```suggestion
1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases/tag/v0.78.0) or later with the following environment variables:
```

I assume this should have been a direct link to `0.78.0`?

File: docs/developer/debugging/host.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ import TabItem from "@theme/TabItem";
 
 Host troubleshooting largely revolves around checking host logs and determining the issue from there. With wasmCloud we aim to log relevant information so that you can diagnose the source of the problem.
 
-The primary wasmCloud host runtime is the [wasmcloud-otp](https://github.com/wasmCloud/wasmcloud-otp) host, and this is what you'll be using if you're running the host from the release tarball or with the [wasmcloud_host](https://hub.docker.com/repository/docker/wasmcloud/wasmcloud_host) docker image. The Javascript host has different log messages and it's recommended to create an issue [here](https://github.com/wasmCloud/wasmcloud-js/issues/new) if you are running into errors in the browser.
+The primary wasmCloud host runtime is the [wasmCloud](https://github.com/wasmCloud/wasmCloud/tree/main/crates/host) host, and this is what you'll be using if you're running the host from the release tarball or with the [wasmcloud](https://hub.docker.com/r/wasmcloud/wasmcloud) docker image. The Javascript host has different log messages and it's recommended to create an issue [here](https://github.com/wasmCloud/wasmcloud-js/issues/new) if you are running into errors in the browser.
```
Comment (vados-cosmonic) [inline docs/developer/debugging/host.md]:
```suggestion
The primary wasmCloud host runtime is [`wasmcloud-host`](https://github.com/wasmCloud/wasmCloud/tree/main/crates/host), which you'll be using if you're running the host from the release tarball or with the [wasmcloud](https://hub.docker.com/r/wasmcloud/wasmcloud) docker image. The Javascript host has different log messages and it's recommended to create an issue [here](https://github.com/wasmCloud/wasmcloud-js/issues/new) if you are running into errors in the browser.
```

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket:

Some nits but can be addressed in a follow up

**Review by brooksmtownsend — APPROVED** [review]

File: docs/deployment/tracing.md (hunk 1 above)
Comment (connorsmith256) [reply docs/deployment/tracing.md]: I figured linking to the general release page would make this link more perma-y. I don't want to have to update this again

And now that I mention it, I think removing the version entirely makes more sense. We've had tracing support for like 18 months so no need to specify a version

Comment (lachieh) [reply docs/deployment/tracing.md]:
```diff
- 1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases) or later with the following environment variables:
+ 1. wasmCloud [v0.78.0 or later](https://github.com/wasmCloud/wasmCloud/releases) with the following environment variables:
```
~could do this~ nevermind

**Pull request merged**